Tools/WebKitTestRunner/cocoa/TestControllerCocoa.mm

```diff
@@ -152,6 +199,45 @@ void initializeWebViewConfiguration(const char* libraryPath, WKStringRef injecte
         reinterpret_cast<IMP>(swizzledProcessImageAnalysisRequest)
     );
 #endif
+
+#if PLATFORM(MAC)
+    // See NetworkProcess::platformInitializeNetworkProcessCocoa for supporting a local DNS resolver on iOS Simulator.
+
+    if (!options.useLocalDNSResolver)
+        return;
+
+    NSUUID *agentUUID = [[NSUUID alloc] init];
```

Comment:
`auto agentUUID = adoptNS([[NSUUID alloc] init]);`

Otherwise, you are leaking.